src/Chapter1.hs

```diff
@@ -542,7 +543,11 @@ value after "=" where the condition is true.
 Casual reminder about adding top-level type signatures for all functions :)
 -}
 
-mid x y z = error "mid: not implemented!"
+mid :: Int -> Int -> Int -> Int
+mid x y z
+    | ((y < x) && (x < z)) || ((z < x) && (x < y)) = x
```

Comment:
Some `()` are redundant here and can be removed. Even though, you can remove all `()` in this code, and it will work the same because of different priorities for `&&` and `||` operators. But it's okay to keep some parenthesis to maintain code readable, though, sometimes some of then can be redundant indeed :slightly_smiling_face: 
```suggestion
    | (y < x && x < z) || (z < x && x < y) = x
```